src/App.vue

```diff
@@ -9,6 +9,9 @@ export default {
 </script>
 
 <style>
+html {
+  overflow-y: scroll;
```

Comment:
Interesting. What does this do?

Reply:
it forces the vertical scroll bar to always be visible. It's disabled if the content all fits in the viewport. 

Before this change, if you had a small enough browser window that there was a scroll bar on the happ list page then when you filtered the happ list and didn't need the scroll bar anymore, the scroll bar would disappear and the content jumped slightly to the right to fill the space.

This makes it so the scroll bar is always there so there's no jump. I don't love it, but @SirRobLyon prefers it to the jumping, and this is the standard cross browser solution for that kind of jumping. The alternative would be to get too clever by far.